src/dbus.c

```diff
@@ -163,13 +167,15 @@ DBUS_METHOD(dunst_NotificationAction);
 DBUS_METHOD(dunst_NotificationCloseAll);
 DBUS_METHOD(dunst_NotificationCloseLast);
 DBUS_METHOD(dunst_NotificationShow);
+DBUS_METHOD(dunst_RuleEnable);
 DBUS_METHOD(dunst_Ping);
 static struct dbus_method methods_dunst[] = {
         {"ContextMenuCall",        dbus_cb_dunst_ContextMenuCall},
         {"NotificationAction",     dbus_cb_dunst_NotificationAction},
         {"NotificationCloseAll",   dbus_cb_dunst_NotificationCloseAll},
         {"NotificationCloseLast",  dbus_cb_dunst_NotificationCloseLast},
         {"NotificationShow",       dbus_cb_dunst_NotificationShow},
+        {"RuleEnable",             dbus_cb_dunst_RuleEnable},
```

Comment:
This list should be sorted apparently (there is a test about it that failed). You can test on your own system by running `make test`.

Reply:
Ups, I didn't noticed, thank you. Fixed with https://github.com/dunst-project/dunst/pull/981/commits/f4e445b7f6c4df6e907a4f9bdf597ca24ca696e2.

Reply:
Great